src/useCases/getAccountBeacons.ts

```diff
@@ -0,0 +1,16 @@
+import { IBeacon } from "../entities/beacon";
+import { IAppContainer } from "../lib/appContainer";
+
+export type GetAccountBeaconsFn = (accountId: string) => Promise<IBeacon[]>;
+
+export const getAccountBeacons =
+  ({
+    getAccessToken,
+    accountHolderApiGateway,
+  }: IAppContainer): GetAccountBeaconsFn =>
+  async (accountId: string) => {
+    const accessToken = await getAccessToken();
+    const accountHolderDetails =
+      await accountHolderApiGateway.getAccountBeacons(accountId, accessToken);
```

Comment:
Similarly, it would be worth updating the name of this method to match the name of this use case